recipes/pyhams/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set version = "1.2.1" %}
+
+package:
+  name: pyhams
+  version: {{ version }}
+
+source:
+  url: https://github.com/WISDEM/pyHAMS/archive/v{{ version }}.tar.gz
+  sha256: 7d72a30cfb13be162c5aa2cea0e053ea82f5b0f638242f66452dc0dab454678e
+  # Download the tar-ball and do: openssl sha256 file.tar.gz to update
+
+build:
+  number: 0
+  skip: true  # [py<39]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

```suggestion
  script:
    - {{ PYTHON }} -m pip install . --no-deps -vv
    - rm -rf {{ SP_DIR }}/meson_build       # [unix]
    - rmdir /s /y {{ SP_DIR }}/meson_build  # [win]
```
You're still installing some extra stuff into site packages.

---

That is a great suggestion in the script!